Cargo.toml

```diff
@@ -1,5 +1,21 @@
 [workspace]
 members = [
+    "crates/bdk",
+    "crates/chain",
+    "crates/file_store",
+    "crates/electrum",
+    "example-crates/keychain_tracker_electrum",
+    "example-crates/keychain_tracker_esplora",
+    "example-crates/keychain_tracker_example_cli",
+    "example-crates/wallet_electrum",
+    "example-crates/wallet_esplora",
+    "example-crates/wallet_esplora_async",
+    "example-crates/wallet_esplora_tor",
+    "nursery/tmp_plan",
+    "nursery/coin_select"
+]
+
+default-members = [
```

Comment:
Why do we need both members and default-members?

Reply:
To prevent `example-crates/wallet_esplora_tor` to be built by default. It depends on `libtor`, which requires a C compiler, but BDK as a library doesn't and most important it shouldn't.

In this configuration one can build BDK without installing the additional dependencies simply by running
```bash
cargo build
```

And after installing the additional dependencies one can build this example with
```bash
cargo build -p bdk-esplora-wallet-tor-example
```

I agree, this is not the most elegant solution, so I'm open for suggestions how to improve it.